src/main/resources/templates/home.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <title>Zarządzanie zadaniami</title>
+    <link rel="stylesheet" href='../static/style.css' th:href="@{style.css}">
+</head>
+<body>
+<div class="container-xxl" id="banner">
+    <h1>Zarządzanie zadaniami</h1>
+    <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+        <li class="nav-item" role="presentation">
+            <a href="/?task=completed">
+                <button class="nav-link active" id="pills-profile-tab" data-bs-toggle="pill"
+                        data-bs-target="#pills-profile" type="button" role="tab" aria-controls="pills-profile"
+                        aria-selected="false">Zadania zrealizowane
+                </button>
+            </a>
+        </li>
+        <li class="nav-item" role="presentation">
+            <a href="/">
+                <button class="nav-link active" id="pills-contact-tab" data-bs-toggle="pill"
+                        data-bs-target="#pills-contact" type="button" role="tab" aria-controls="pills-contact"
+                        aria-selected="false">Zadania niezrealizowane
+                </button>
+            </a>
+        </li>
+        <li class="nav-item" role="presentation">
+            <a href="/?task=sortByDateAsc">
+                <button class="nav-link active" id="pills-contact-tab" data-bs-toggle="pill"
+                        data-bs-target="#pills-contact" type="button" role="tab" aria-controls="pills-contact"
+                        aria-selected="false">Wyświetl zadania z najbliższym terminem
+                </button>
+            </a>
+        </li>
+    </ul>
+    <br>
+</div>
+</div>
+<div class="container-xxl" id="tasks">
+    <table class="table table-striped table-hover">
+        <tr>
+            <th> Nazwa zadania</th>
+            <th> Opis zadania</th>
+            <th> Termin do oddania</th>
+            <th> Oznacz jako zakończone</th>
+            <th> Edycja</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr th:if="${tasks.empty}">
+            <td colspan="5"> Brak zadań</td>
+        </tr>
+        <tr th:each="task : ${tasks}">
+            <td><span th:text="${task.name}"> Nazwa </span></td>
+            <td><span th:text="${task.description}"> Opis </span></td>
+            <td><span th:text="${task.deadLine}"> Data graniczna </span></td>
+            <td>
+                <a th:href="@{|/markAsComplete?id=${task.id}|}">
+                    <button th:if="${task.isFinish()} == false" th:field="${taskModify.setId(task.id)}"
+                            type="submit" class="btn btn-primary">Oznacz
+                    </button>
+                </a>
```

Comment:
Fragment `th:field="${taskModify.setId(task.id)}"` jest zbędny, w `th:if="${task.isFinish()} == false"` staraj się używać raczej zapisu `th:if="${not task.isFinish()}"`